src/err/mod.rs

```diff
@@ -448,7 +448,7 @@ impl PyErr {
 
     /// Returns true if the current exception is instance of `T`.
     #[inline]
-    pub fn is_instance(&self, py: Python<'_>, typ: &PyType) -> bool {
+    pub fn is_instance(&self, py: Python<'_>, typ: &PyAny) -> bool {
```

Comment:
```suggestion
    pub fn is_instance(&self, py: Python<'_>, ty: &PyAny) -> bool {
```
For consistency with `PyAny::is_instance`